docs/src/main/asciidoc/scheduler-reference.adoc

```diff
@@ -207,6 +207,8 @@ class MyJobs {
 }
 ----
 
+NOTE: By default, the scheduler is not started unless a `@Scheduled` business method is found. You may need to force the start of the scheduler for "pure" programmatic scheduling. See also <<quartz.adoc#quartz-configuration-reference,Quartz Configuration Reference>>.
```

Comment:
I'm not entirely sure how this link will be rendered in the HTML. We need it to end up being `quartz` without any extension.

Could you check?

Reply:
It renders just fine. Actually, it's the only way I found to link a section from a different document. See https://asciidoctor.org/docs/user-manual/#inter-document-cross-references.

Reply:
My fear was that it renders with `.html` at the end which would not help with SEO.

Reply:
Yeah and that's exactly what happens unfortunately.

Reply:
I think I fixed the issue here: https://github.com/quarkusio/quarkusio.github.io/pull/661 .

But if another pair of eyes can check that I didn't break something else, that would be welcome!